packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts

```diff
@@ -157,15 +157,16 @@ export const getDonationList = async (walletList: WalletTypes[]) => {
 export const getFootprintList = async (walletList: WalletTypes[]) => {
     const promises = walletList.map(({ address, platform }) => {
         return RSS3.getFootprints(formatEthereumAddress(address)).then((result) => {
+            console.log({ result })
             if (result) {
                 return {
                     address,
                     collections: result?.map((asset) => ({
                         key: asset?.id,
                         address: asset?.id,
                         platform: platform ?? NetworkPluginID.PLUGIN_EVM,
-                        iconURL: asset?.detail?.image_url ?? undefined,
-                        name: asset?.detail?.name,
+                        iconURL: asset?.imageURL,
```

Comment:
`result` has already been checked above, it's not nullable. `assets` is `RSS3BaseAPI.Donation`, it's not nullable either.